README.md

```diff
@@ -488,7 +490,7 @@ ull}}]],
 ```
 </details>
 
-## `RegisterClient`
+## <a name="registerclient"></a>`RegisterClient`
```

Comment:
I may be wrong here, but do we not need to use the `id` attribute to create links within the doc?

Reply:
I think I've gotten in the bad habit of not using it because the tech docs tool takes care of it for me ...

Reply:
If you want the anchors to be portable outside of Github, you need to be explicit. The style of using `name` instead of `id` and an empty anchor rather than wrapping the text is a mix of old habits and taste. Mine is

```markdown
## <a id="registerclient">`RegisterClient`</a>
```

Reply:
Noted, thanks @arnau. Will change throughout.